strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/layout/npm/NpmUser.java

```diff
@@ -0,0 +1,95 @@
+package org.carlspring.strongbox.controllers.layout.npm;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSetter;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class NpmUser
+{
+
+    private String id;
+
+    private String name;
+
+    private String password;
+
+    private String type;
+
+    private String[] roles;
+
+    private String date;
+
+    private String email;
+
+    @JsonCreator
+    public NpmUser() {}
+
+    @JsonCreator
+    public NpmUser(@JsonProperty(value = "_id", required = true) String id,
+                   @JsonProperty(value = "name", required = true) String name,
+                   @JsonProperty(value = "password", required = true) String password,
+                   @JsonProperty(value = "type", required = true) String type,
+                   @JsonProperty(value = "roles", required = true) String[] roles,
+                   @JsonProperty(value = "date", required = true) String date,
+                   @JsonProperty(value = "email") String email)
+    {
+        this.id = id;
+        this.name = name;
+        this.password = password;
+        this.type = type;
+        this.roles = roles;
+        this.date = date;
+        this.email = email;
+    }
+
+    public String getName() {
```

Comment:
Cosmetic: Could you please follow the project's [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html)? Thanks! :)

Reply:
This is the hardest one to follow hahah, but yes

Reply:
If you have IntelliJ, we have the codestyle as an importable file. We also have it for Eclipse, but it's not half as thorough...

Reply:
I have it installed. When I went to reformat, it changed tons of files

Reply:
Strange...

Reply:
Seems like its working for individual files